src/main/java/com/amazon/ion/impl/IonReaderContinuableApplicationBinary.java

```diff
@@ -516,7 +533,8 @@ protected void restoreSymbolTable(SymbolTable symbolTable) {
             // Note: this will only happen when `symbolTable` is the system symbol table.
             resetSymbolTable();
             cachedReadOnlySymbolTable = symbolTable;
-            resetImports();
+            // FIXME: This should take into account the version at the point in the stream.
+            resetImports(1, 0);
```

Comment:
What's stopping us from passing in the real major and minor version here?

Reply:
With the current implementation, the version of the symbol table is determined by the system symbol table that it imports. In Ion 1.1, the local symbol table does not necessarily import any system table. I think there's a lot more work that will need to be done in order to get Ion 1.1 and the current `SymbolTable` API to work together.